app/services/SubmissionService.scala

```diff
@@ -55,12 +56,11 @@ class SubmissionService @Inject()(
     auditService.auditAllPagesUserInput(getCachedData(exportsCacheModel))
     exportsConnector.submitExportDeclaration(data.ducr, data.lrn, data.payload).flatMap {
       case HttpResponse(ACCEPTED, _, _, _) =>
-        cacheService.remove(cacheId).map { _ =>
-          auditService.audit(AuditTypes.Submission, auditData(data.lrn, data.ducr, Success.toString))
-          exportsMetrics.incrementCounter(submissionMetric)
-          timerContext.stop()
-          data.lrn
-        }
+        cacheService.remove(sessionId)
```

Comment:
Here we have little change of semantic - what if in removing from cache there goes something wrong - we don't have even know abut it.

Reply:
Have changed how we do this so all code in controller is handled inside the future.